services/src/main/java/org/exoplatform/addons/gamification/service/AnnouncementService.java

```diff
@@ -59,6 +59,16 @@ Announcement createAnnouncement(Announcement announcement, Map<String, String> t
    */
   Announcement updateAnnouncement(Announcement announcement, boolean broadcast) throws ObjectNotFoundException;
 
+  /**
+   * Cancel announcement
+   *
+   * @param announcementId technical identifier of a challenge
+   * @return a {@link Announcement} Object
+   * @throws ObjectNotFoundException when the announcement identified by its
+   *           technical identifier is not found
+   */
+  Announcement cancelAnnouncement(Long announcementId, String username) throws ObjectNotFoundException, IllegalAccessException;
```

Comment:
Better using CRUD names in API.

```suggestion
  Announcement deleteAnnouncement(Long announcementId, String username) throws ObjectNotFoundException, IllegalAccessException;
```